tests/src/test/scala/whisk/core/database/test/ReplicatorTests.scala

```diff
@@ -209,20 +208,23 @@ class ReplicatorTests extends FlatSpec
 
```

Comment:
Is it possible that the `putDoc` fails with an exception - for example, due to timeout? If yes, the database won't be removed.